src/concrete/ml/sklearn/base.py

```diff
@@ -65,7 +68,15 @@
 )
 from ..torch import NumpyModule
 from .qnn_module import SparseQuantNeuralNetwork
-from .tree_to_numpy import tree_to_numpy
+from .tree_to_numpy import (
+    _compute_lsb_to_remove_for_trees,
+    get_equivalent_numpy_forward_from_onnx_tree,
+    get_onnx_model,
+    is_regressor_or_partial_regressor,
+    preprocess_tree_predictions,
+    tree_onnx_graph_preprocessing,
+    tree_to_numpy,
+)
```

Comment:
Can we keep using all these within tree_to_numpy?

Reply:
Moved everything to tree_to_numpy.